### DISCUSSION TITLE
<title>sharing the code

### DISCUSSION THREAD

**Discussion by MercaCityNet** [community/community#39928] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Question

### Body

Hello,
I am new at github and I will use it mostly from the point of view of project management.
I have no knowledge about programming - so thank you in advance for your understanding :)
I might need some advices or best practice tipps:

-  e.g. If I invite a person to repository and grant them the role "read" does the person can still copy and change the code? is there something I shall consider?
- how can I share just a part of the code for reading?

Thank you and best regards!

Ag

**Comment by MitkoVtori** [comment, marked as answer]
If the repo is private, you can just make a public one and copy-paste the code, that you want to show. If you want a contributor, here is how to manage access for your contributors: [docs](https://docs.github.com/en/account-and-profile/setting-up-and-managing-your-github-profile/managing-contribution-settings-on-your-profile) 
Else, you can see more about the roles [here](https://docs.github.com/en/organizations/managing-user-access-to-your-organizations-repositories/repository-roles-for-an-organization)

**Reply by MercaCityNet** [reply]
thank you!

**Comment by ThinuraNaijar** [comment]
can you help me to make the car webpage
